### DISCUSSION TITLE
Default key mappings of core plugin Project different or not available in Lunarvim?

### DISCUSSION THREAD

**Discussion by onno-schuit** [LunarVim/LunarVim#2806] [Q&A] [answered]
The documentation for the Project plugin (ahmedkhalf/project.nvim) states: "project.nvim comes with the following mappings". But none of the mappings seem to work, as far as I can tell, in Lunarvim.

As an example, if I type in 'f' in normal mode (for find_project_files), absolutely nothing happens.

Sorry if I'm overlooking something. I've tried to google this issue and looked at the packer_compiled.lua file, but I couldn't find any clues.

**Comment by rebuilt** [comment, marked as answer]
Reading the keybinding [section](https://github.com/ahmedkhalf/project.nvim#telescope-mappings) leads me to believe they are meant to be used on the telescope screen.
Run
```
:Telescope projects
```
And try the keybindings there.

**Reply by onno-schuit** [reply]
Ah, of course (slaps forehead). Thanks a bunch!
